_episodes/15-conditionals.md

```diff
@@ -49,7 +49,7 @@ FIXME: lesson content.
 > for filename in glob.glob('data/*.csv'):
 >     contents = pd.read_csv(filename)
 >     ____:
->         FIXME: what to include as processing step?
+>         print filename,len(contents)
```

Comment:
+1 on the change, but we're using [Python][1] [3][2], so `print(filename, len(contents))`.

[1]: https://github.com/swcarpentry/python-novice-gapminder/blob/ca400f197039a50ff280b92bdfd68ba4ad90a47a/_extras/design.md#lists
[2]: https://github.com/swcarpentry/workshop-template/blob/e39365cb9a9bf8186435f92810cc254a2cc914c7/index.html#L620